packages/vuetify/src/components/VAlert/__tests__/VAlert.spec.ts

```diff
@@ -89,26 +92,22 @@ describe('VAlert.ts', () => {
     expect(wrapper.contains('.v-icon')).toBe(false)
   })
 
-  // TODO: this fails without sync, nextTick doesn't help
-  // https://github.com/vuejs/vue-test-utils/issues/1130
-  it.skip('should display contextual colors by type', async () => {
+  it('should display contextual colors by type', () => {
     const wrapper = mountFunction({
       propsData: { type: 'error' },
     })
 
     expect(wrapper.classes('error')).toBe(true)
+  })
 
-    wrapper.setProps({ type: 'success' })
-    await wrapper.vm.$nextTick()
-    expect(wrapper.classes('success')).toBe(true)
-
-    wrapper.setProps({ type: 'warning' })
-    await wrapper.vm.$nextTick()
-    expect(wrapper.classes('warning')).toBe(true)
+  ;['success', 'error', 'warning', 'info'].forEach(type => {
```

Comment:
can avoid using `setProps` and just test all four colors using a simple for loop - no `await nextTick` is needed here, seems legit